crates/sui-move/src/prove.rs

```diff
@@ -92,6 +93,7 @@ impl Prover {
             ),
         );
 
+        eprintln!("WARNING: the level of Move Prover support for Sui is currently limited; use at your own risk");
```

Comment:
The idea makes sense but I am worried that this will change in time. Actually, I am not 100% confident what what does and does not work at this point and I feel like a more generic message would be sufficient to trigger further investigation on the side of a person that still wants to go for it despite the warning.

Reply:
Maybe then we could just explain out what "at your own risk" entails? Like just explaining: "Not everything is guaranteed to work. Please file an issue if an update breaks your usage" 
Something to that effect. Like I want people to know that it might not work, but we should also be receptive if we break something that was working

Reply:
I finessed the warning message a bit but in reality we don't have enough resources to encourage people to file issue so I did not include that part...